src/app/users/request-phone-code.ts

```diff
@@ -15,6 +14,9 @@ import { PhoneCodesRepository } from "@services/mongoose/phone-code"
 import { RedisRateLimitService } from "@services/rate-limit"
 import { TwilioClient } from "@services/twilio"
 
+const randomIntFromInterval = (min, max) =>
```

Comment:
Could also go into utils

Reply:
thought about it but really just used in `request-phone-code.ts`. not sure function is cryptographically secured for cryptographic stuff so don't want to expose that broadly either.